features/layout/footer/footer.tsx

```diff
@@ -0,0 +1,38 @@
+import styles from "./footer.module.scss";
+import packageJson from "../../../package.json";
+
+export function Footer() {
+  const appVersion = packageJson.version;
+
+  return (
+    <footer className={styles.container}>
+      <ul className={styles.linkList}>
+        <li>
+          <a className={styles.anchor} href="#">
+            Docs
+          </a>
+        </li>
+        <li>
+          <a className={styles.anchor} href="#">
+            API
+          </a>
+        </li>
+        <li>
+          <a className={styles.anchor} href="#">
+            Help
+          </a>
+        </li>
+        <li>
+          <a className={styles.anchor} href="#">
```

Comment:
🌵 Maybe jump ahead of the requirements a little and use the `<Link />` component?